File: find-unique-binary-string.ts
{
    function findDifferentBinaryString(nums: string[]): string {
        let r = '';

        for (let i = 0; i < nums.length; i++) {
            r += nums[i][i] === '1' ? '0' : '1';        
        }


        return r;
    };

    console.log(findDifferentBinaryString(["00","01"]))
}

File: find-unique-binary-string.rs
pub fn find_different_binary_string(nums: Vec<String>) -> String {
    let mut r = String::new();

    for i in 0..nums.len() {
        let chars: Vec<char> = nums[i].chars().collect(); 
        
        if chars[i] == '1' {
            r.push('0');
        } else {
            r.push('1');
        }
    }

    r
}